src/pages/ThorChainLP/components/AddLiquidity/AddLiquidityInput.tsx

```diff
@@ -431,11 +432,12 @@ export const AddLiquidityInput: React.FC<AddLiquidityInputProps> = ({
   // - when routed from "Your Positions" where an active opportunity was found from the RUNE or asset address, but the wallet
   // doesn't support one of the two
   const walletSupportsOpportunity = useMemo(() => {
+    if (isDemoWallet) return false
```

Comment:
This will need matching handling in `maybeOpportunityNotSupportedExplainer` to avoid the empty alert div

Reply:
done in [d1a2eaf](https://github.com/shapeshift/web/pull/7269/commits/d1a2eaf5cc280a650761a53eba9eb72182f9a6a1)